email_flood_monitoring.Rmd

```diff
@@ -0,0 +1,127 @@
+---
+title: "email"
+output: blastula::blastula_email
+---
+
+<style type = "text/css">
+
+h1, h2, h3 {
+  font-family: Arvo, "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: normal
+}
+
+p {
+  font-family: "Source Sans Pro", "Helvetica Neue", Helvetica, Arial, sans-serif
+}
+
+</style>
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = FALSE,
+  warning = FALSE,
+  message = FALSE,
+  fontfamily = "Sans Source Pro",
+  results="asis",
+  fig.align='center',
+  fig.cap = "" 
+)
+```
+
+```{r}
+# library(targets)
+# library(tidyverse)
+# # library(dplyr)
+# # library(ggplot2)
+# library(glue)
+# library(gghdx)
+# # library(readr)
+# library(googledrive)
+# library(googlesheets4)
+# library(blastula)
+# library(here)
+# library(sf)
+# library(janitor)
+gghdx()
+source("src/email/email_utils.R")
+```
+
+```{r, class.output = "banner"}
+banner_drib <- drive_dribble %>%
+  filter(name == "centre_banner.png")
+drive_download(banner_drib, path = banner_path <- tempfile(fileext = ".png"))
+
+add_image(
+  file = banner_path,
+  alt = "Centre for Humanitarian Data banner",
+  align = "left",
+  width = "25%"
+)
+```
+
+
+# Nigeria Riverine Flood Monitoring
+
+The Nigeria riverine flood monitoring is a pilot project managed by the OCHA Centre for Humanitarian Data in collaboration with the Google Flood Forecasting Initiative in order to bring attention to changes in key indicators relevant to humanitarian response. The monitoring is currently limited to the Niger and Benue river basins in Nigeria.
+
+## Flood Status `r dt_made_chr`
+
+**`r txt_warning_status`**
```

Comment:
Something about this set up I don't like, which is that the warning is not easy to see since it's in smaller text. Could it potentially just be a single ## header?

```markdown
## `r dt_made_chr`: Flood warning/No flood warning
```

Reply:
yeah I agree and I also don't like how it says `Flood Warning - Date` in big and then below says `No warning`. Your suggestion makes sense to me - will adjust